python/src/remotebmi/client/client.py

```diff
@@ -7,14 +7,21 @@
 
 
 class RemoteBmiClient(Bmi):
-    def __init__(self, base_url, timeout=60 * 60 * 24, max_keepalive_connections=0):
+    def __init__(
+        self,
+        base_url,
+        timeout=60 * 60 * 24,
+        max_keepalive_connections=0,
+        client: Client | None = None,
+    ):
         """RemoteBmiClient constructor
 
         Args:
             base_url: Where the remote BMI server is running.
             timeout: How long a response can take.
                 Defaults to 1 day. Set to None to disable timeout.
             max_keepalive_connections: How many connections to keep alive.
```

Comment:
Could you elaborate on the meaning of this kwarg? I don't think this makes sense for users. Do we need to expose this at all?

Reply:
the R server does not like reusing same connection